### DISCUSSION TITLE
`q-select` built-in simple filter

### DISCUSSION THREAD

**Discussion by robsontenorio** [quasarframework/quasar#11773] [Ideas / Proposals]
Currently any kind of filter for q-select must be implemented manually.

For the simple cases (client side only) why not provide the basic filter out of box? This would cut a lot of boilerplate.

**Comment by pdanpdan** [comment]
That would only save a few letters that are the name of the function.
If you use the same filter function multiple times put it in a file and import it

**Comment by npokr** [comment]
Calling such a function would be rather cumbersome. It is necessary to pass 1) unfiltered options (which will have to be returned from setup) 2) options, and 3) option-label.
I tried to package such filtering in a simple wrapper, but found no response when options were changed: 
https://github.com/quasarframework/quasar/discussions/11846
https://codepen.io/npokr/pen/OJxQELm
How do I change the options in the wrapped QSelect?

**Reply by metalsadman** [reply]
see my answer in your thread https://github.com/quasarframework/quasar/discussions/11846
